### ISSUE TITLE
Refreshing sideloaded app takes forever

### ISSUE THREAD

**Issue by abdullahaltun70** [altstoreio/AltStore#1136]
After 7 days if I install Altstore again a previously sideloaded app takes forever to refresh. It can take up to 30 min for one app, where altstore only takes 20 sec. Sometimes it doesn't even refresh whatsoever. Any tips or known issues?

**Comment by GeardoGrunt** [comment]
Having the same exact issue myself. I updated from Windows 10 to Windows 11 (ReviOS). Would you happen to also upgrade your Windows version? 
Have you fixed the issue in the meantime? If so I'd love to hear how you did it!

**Comment by GeardoGrunt** [comment]
I fixed it. For anyone having this issue, where's what I did:
1. Uninstall all custom apps including AppStore
2. Go to iTunes and disable "Sync with this iPhone over Wi-fi" (might actually not do anything but it's what I did anyway)
3. Connect iPhone via USB and install Altstore
4. Re-enable "Sync with this iPhone over Wi-fi" on iTunes
5. Disconnect iPhone from USB and try to refresh the apps and you should be good

Let me know if it worked for you.

**Comment by abdullahaltun70** [comment]
> I fixed it. For anyone having this issue, where's what I did:
> 
> 1. Uninstall all custom apps including AppStore
> 2. Go to iTunes and disable "Sync with this iPhone over Wi-fi" (might actually not do anything but it's what I did anyway)
> 3. Connect iPhone via USB and install Altstore
> 4. Re-enable "Sync with this iPhone over Wi-fi" on iTunes
> 5. Disconnect iPhone from USB and try to refresh the apps and you should be good
> 
> Let me know if it worked for you.

Yes This worked for me! I deleted all the apps and altstore. Afterwards every new sideload worked perfect!

**Comment by jiznon** [comment]
Thanks! This also worked for me!

**Comment by jul-reb** [comment]
I found that manually starting iTunes and logging in there when prompted fixed the issue :)
Perhaps iTunes is used in the process?
A descriptive message with regards to refresh status/actions would make troubleshooting a lot of a smoother experience.
